src/main/java/com/darkona/adventurebackpack/util/EnchUtils.java

```diff
@@ -40,6 +53,18 @@ public static boolean isSoulBounded(ItemStack stack) {
         return false;
     }
 
+    public static boolean isTranslucent(ItemStack stack) {
+        NBTTagList stackEnch = stack.getEnchantmentTagList();
+        if (TRANSLUCENCY_ID >= 0 && stackEnch != null) {
+            for (int i = 0; i < stackEnch.tagCount(); i++) {
+                int id = stackEnch.getCompoundTagAt(i).getInteger("id");
+                int lvl = stackEnch.getCompoundTagAt(i).getInteger("lvl");
+                if (id == TRANSLUCENCY_ID && lvl == 2) return true;
+            }
+        }
```

Comment:
Please, use `EnchantmentHelper` instead of working with NBT directly. Also rework all other methods in this class to use it.

Reply:
Fixed in https://github.com/GTNewHorizons/AdventureBackpack2/pull/23/commits/4c19d41cc9b6cc409501158abbe98f689548e3ea